examples/Bremsstrahlung/pymodules/energyHistogram.py

```diff
@@ -0,0 +1,227 @@
+import sys
```

Comment:
- [ ] missing copyright info and completely taken from my wiki example
- [ ] generic module and not specific to this example, put it in `src/tools/share/python/` instead